### ISSUE TITLE
Get certificate notbefore, notafter properties

### ISSUE THREAD

**Issue by carxwol** [wkeeling/selenium-wire#319]
Hi there,

I discovered this selenium extension recently and it's very useful, thank you!

But i try to understand how to get notbefore and notafter properties i see in seleniumwire/thirdparty/mitmproxy/certs.py (line 435  and 440).

The only way i found, after introspecting request and response objects, even with interceptors, reading some code to see what i can customize is to patch InterceptRequestHandler._create_response method like this.

```
diff --git a/seleniumwire/handler.py b/seleniumwire/handler.py
index 49d286f..936d1ce 100644
--- a/seleniumwire/handler.py
+++ b/seleniumwire/handler.py
@@ -153,6 +153,8 @@ class InterceptRequestHandler:
                 key=cert.keyinfo,
                 signature_algorithm=cert.x509.get_signature_algorithm(),
                 expired=cert.has_expired,
+                notbefore=cert.notbefore,
+                notafter=cert.notafter,
                 issuer=cert.issuer
             )
```

Is there a simpler solution ?

Regards,
Stéphane alias carxwol

**Comment by wkeeling** [comment]
Thanks for raising this.

It looks like we need to add these properties and perhaps some other properties to the certificate dictionary that Selenium Wire exposes. In the meantime patching the `_create_response()` is probably the best workaround. I'll look at adding `notbefore` and `notafter` as well as the other missing properties ASAP.

**Comment by wkeeling** [comment]
These properties are now available in 4.3.1

**Comment by carxwol** [comment]
Thank you very much!
